examples/file-sharing.rs

```diff
@@ -351,9 +351,9 @@ mod network {
         }
 
         /// Respond with the provided file content to the given request.
-        pub async fn respond_file(&mut self, file: String, channel: ResponseChannel<FileResponse>) {
+        pub async fn respond_file(&mut self, file: &Vec<u8>, channel: ResponseChannel<FileResponse>) {
             self.sender
-                .send(Command::RespondFile { file, channel })
+                .send(Command::RespondFile { file: file.to_vec(), channel })
```

Comment:
Nit: I think the reference on line 354 is unnecessary if we are going to create a copy anyway and it will remove the diff on line 356.

Reply:
OK, done. Thanks~